instrumentation/opencensus-shim/javaagent/src/main/java/io/opentelemetry/opencensusshim/OpencensusShimInstrumentationModule.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class OpencensusShimInstrumentationModule extends InstrumentationModule {
+
+  public OpencensusShimInstrumentationModule() {
+    super("opencensus-shim", "io.opentelemetry.opencensusshim");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Arrays.asList(new OpenTelemetryCtxInstrumentation());
```

Comment:
Collections.singletonList, which we usually static import (see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports)
```suggestion
    return singletonList(new OpenTelemetryCtxInstrumentation());
```